app/page.tsx

```diff
@@ -1,10 +1,11 @@
 import Image from 'next/image'
 import { Button } from '@/components/UI/button'
+import CountryInfo from '@/components/countryInfo'
 
 export default function Home() {
   return (
     <main className="">
-    
+      <CountryInfo />
     </main>
   )
 }
```

Comment:
- [ ] Great work, @NurkaAmre! I appreciate the effective rendering of Country info on the home page. However, there's a minor issue that needs attention. It seems you've already wrapped your body elements with the `main` tag, which is excellent. Just a quick reminder, it's a common and good practice to use it one page at a time.

![Capture](https://github.com/NurkaAmre/nomad/assets/35267447/5856e87e-2e41-4eb2-867f-c59f62fd6472)

***To address this, you can replace the `main` tag with a `React Fragment` or a `div` tag.***